src/js/tabs/configuration.js

```diff
@@ -518,6 +518,11 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
         if (semver.gte(CONFIG.apiVersion, "1.31.0")) {
             $('input[name="fpvCamAngleDegrees"]').val(RX_CONFIG.fpvCamAngleDegrees);
+            if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
+                $('input[name="fpvCamAngleDegrees"]').attr({
+                    "max" : 90
+                });
```

Comment:
Maybe is simpler to make `.attr("max", "90")` when there is only one attr to change. but is good.

Reply:
Done.